src/ext/java/org/opentripplanner/ext/fares/impl/OrcaFareServiceImpl.java

```diff
@@ -0,0 +1,665 @@
+package org.opentripplanner.ext.fares.impl;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.util.Collection;
+import java.util.Currency;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import org.opentripplanner.ext.fares.model.FareContainer;
+import org.opentripplanner.ext.fares.model.FareProduct;
+import org.opentripplanner.ext.fares.model.FareRuleSet;
+import org.opentripplanner.ext.fares.model.RiderCategory;
+import org.opentripplanner.model.plan.Leg;
+import org.opentripplanner.routing.core.FareType;
+import org.opentripplanner.routing.core.ItineraryFares;
+import org.opentripplanner.transit.model.basic.Money;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.transit.model.network.Route;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OrcaFareServiceImpl extends DefaultFareService {
```

Comment:
We have stopped adding the word `Impl` to the end of these calculators as nobody thought that it adds anything useful. Can you please simply call it `OrcaFareService`?